python/Ganga/Utility/files.py

```diff
@@ -12,6 +12,8 @@
 
 def expandfilename(filename, force=False):
     """expand a path or filename in a standard way so that it may contain ~ and ${VAR} strings"""
+    if not filename:
+        return filename
```

Comment:
Then we need to add a __lot__ of protection against this everywhere else and there are mehods passing None vs '' which also causes problems.